cfpb/consumerfinance.gov
# Version Bump for CCDB5-UI

This release includes a number of fixes and improvements for CCDB5-UI including a change in filter logic when parent-level filter elements are selected.

More information about the release is available here: https://github.com/cfpb/ccdb5-ui/releases/tag/2.3.0

## Additions

- None


## Removals

- None


## Changes

- 2.3.0 library for CCDB5-UI


## How to test this PR

1. Open CCDB search
1. Click "Mortgage" product
1. See that all sub-products are selected, but only "Mortgage" has been added as a filter pill to the page

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
